.github/workflows/release.yml

```diff
@@ -48,11 +48,20 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-latest
-          - windows-latest
-          - macos-latest
-
-    runs-on: ${{ matrix.os }}
+          - name: ubuntu-latest
+            cross_compile_os: linux
+            cross_compile_target: aarch64-unknown-linux-gnu
+          - name: ubuntu-latest
+            cross_compile_os:
```

Comment:
Not sure if we need to specify those... If we don't, surely `${{ }}` expressions referencing those fields will fail?

Reply:
Yeah, I don't think we need to specify this. Can't say for sure though.